src/Forms/GridField/GridFieldDetailForm_ItemRequest.php

```diff
@@ -968,7 +968,7 @@ private function getModelName(): string
      * Get Pjax response negotiator so form submission mirrors other form submission in the CMS.
      * See LeftAndMain::getResponseNegotiator()
      */
-    private function getResponseNegotiator(DBHTMLText $renderedForm): PjaxResponseNegotiator
+    private function getResponseNegotiator(string $renderedForm): PjaxResponseNegotiator
```

Comment:
The `DBHtmlText` returned from rendering a template will be coerced to string when it's passed in here.
This change is necessary because we also pass the result of `forTemplate()` into this method, and that's explicitly a string.